staff/alex-martinez/precourse/bingo/bingo-v1/index.js

```diff
@@ -0,0 +1,180 @@
+//acumuladores
+var countBingo = false;
+var tiradas = 0;
+var aciertos = 0;
+var linea1 = 0;
+var linea2 = 0;
+var linea3 = 0;
+var cantadoLinea = false;
+
+
+//bombo
+var Bombo = new Array();
+
+const llenarBombo = ()=>{
+    //llenarmos del 1 al 99
+    for(var i=1; i<100; i++){
+        Bombo.push(i);
+    }
+    return Bombo;
+}
+
+//funcion para sacar uno de los numeros de dentro del bombo
+//aleatoriamente
+const numBombo = ()=>{
+    if(Bombo.length >0){
+        //numero aleatorio entre 1 y 99
+        var nAleatiorio = Math.floor(Math.random()*99)+1;
+    }
+    //variable que almacena el numero aleatorio de la posición del array
+    var numArray = Bombo[nAleatiorio];
+    //se elimina del array del bombo el numero aleatorio obtenido
+    Bombo.splice(nAleatiorio,1);
+
+    return numArray;
+}
+var bola;
+const sacarBola = ()=>{
+    //variable que almacena el numero retornado por la funcion
+    bola = numBombo();
+    //si el numero es indefinido volvemos a tirar()
+    if(bola === undefined){
+        sacarBola();
+    }else{
+        console.log(`¡El ${bola}!`);
+    }
+    return bola;
+}
+
+var carton = new Array();
+//función para generar el carton con numero aleatorios
+const crearCarton = () =>{
+    var numerosCarton = []
+    for(i=0; numerosCarton.length<15;i++){
+        var numero = Math.floor(Math.random() * 99) + 1;
+        if(numerosCarton.indexOf(numero) === -1)
+        numerosCarton.push(numero);
+    }
+    for(i=0;i<15;i++){
+        carton.splice(i,1,numerosCarton[i]);
+    }
+    return carton;
+}
+
+//función para mostrar el carton
+const mostrarCarton = ()=>{
+    console.log(`${carton[0]},${carton[1]},${carton[2]},${carton[3]},${carton[4]},${carton[5]},${carton[6]},${carton[7]},${carton[8]},${carton[9]},${carton[10]},${carton[11]},${carton[12]},${carton[13]},${carton[14]}`);
```

Comment:
Hi Alex! As we are coding on ES5 I would suggest you using var instead of const or let. 
Good code anyway!!!